examples/bri-3/prisma/schema.prisma

```diff
@@ -70,6 +70,7 @@ model Workstep {
   workgroupId    String
   securityPolicy String
   privacyPolicy  String
+  circuitInputsTranslationSchema String?
```

Comment:
why is this a string. It should be a JSON object, no? @ognjenkurtic

Reply:
I didn't see a benefit of having it defined as a JSON object. Also wanted to be consistent with how we store other JSON data (i.e. BpiMerkeTree.tree property). If we see that there is a need to store these as anything other than Strings, we can open a separate issue and tackle it across the codebase. What do you think? @Therecanbeonlyone1969

Reply:
@ognjenkurtic sorry. I vote for JSON object. very easy to define, maintain and use and plenty of library support, plus it is very standard! The circuit input parameters can be of all types ... string, BigInt, boolean, hex etc. so we need that flexibility.

Reply:
maybe we can use `Json` here and open new issue to do it consistently across codebase? but also fine to merge as a string and follow up in other PR

if introducing `Json` is not huge overhead i am up for it (especially since there is built in support in prisma https://www.prisma.io/docs/orm/prisma-client/special-fields-and-types/working-with-json-fields)

Reply:
Ok, will introduce JSON